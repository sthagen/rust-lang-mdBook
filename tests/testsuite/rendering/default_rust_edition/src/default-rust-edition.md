# Chapter 1

```rust
let x = 2021;
```

```rust,edition2021
let x = 2021;
```

```rust,edition2024
let x = 2024;
```
