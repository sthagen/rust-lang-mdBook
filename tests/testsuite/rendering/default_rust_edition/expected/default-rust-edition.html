<h1 id="chapter-1"><a class="header" href="#chapter-1">Chapter 1</a></h1>
<pre class="playground"><code class="language-rust edition2021"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>let x = 2021;
<span class="boring">}</span></code></pre>
<pre class="playground"><code class="language-rust edition2021"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>let x = 2021;
<span class="boring">}</span></code></pre>
<pre class="playground"><code class="language-rust edition2024"><span class="boring">#![allow(unused)]
</span><span class="boring">fn main() {
</span>let x = 2024;
<span class="boring">}</span></code></pre>